src/object/obj_class.c

```diff
@@ -328,3 +356,127 @@ daos_oclass_grp_nr(struct daos_oclass_attr *oc_attr, struct daos_obj_md *md)
 	/* NB: @md is unsupported for now */
 	return oc_attr->ca_grp_nr;
 }
+
+/** a structure to map EC object class to gf tables */
```

Comment:
Please extend GF, I'm pretty sure not everyone know about it

Reply:
OK. The GF is hard to fully explain, but I'll add a few comments.
I'll change the struct a little bit also, because the encode matrix also need to be kept, recovery need to use the encode matrix to generate the decode matrix.